wiki/en/Installation-for-Linux.md

```diff
@@ -67,9 +71,21 @@ Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to conf
 
 Restart JACK for the new settings to take effect
 
+### Configure PipeWire
+
+Jamulus Clients need a JACK server included in PipeWire. It is automatically launched when Jamulus is started.
+However, you need to set your audio interface to the "pro-audio" profile and define PipeWire's "rate" and "quantum" parameters beforehand.
```

Comment:
I don't thing "However" is appropriate -- the style in the earlier section should be followed: be directive.